config/1.0.0/earthdata_varinfo_configuration_schema.json

```diff
@@ -29,175 +29,29 @@
-            "type": "number"
-          },
-          "semi_minor_axis": {
-            "description": "Specifies the length, in metres, of the semi-minor axis of the ellipsoidal figure associated with the geodetic datum and used to approximate the shape of the Earth. Commonly denoted using the symbol b. In the case of a spherical Earth approximation this attribute should be omitted (the preferred option) or else set equal to the value of the semi_major_axis attribute. See also the inverse_flattening attribute.",
-            "type": "number"
-          },
-          "standard_parallel": {
-            "description": "Specifies the line, or lines, of latitude at which the developable map projection surface (plane, cone, or cylinder) touches the reference sphere or ellipsoid used to represent the Earth. Since there is zero scale distortion along a standard parallel it is also referred to as a latitude of true scale. In the situation where a conical developable surface intersects the reference ellipsoid there are two standard parallels, in which case this attribute can be used as a vector to record both latitude values, with the additional convention that the standard parallel nearest the pole (N or S) is provided first. Domain: -90.0 <= standard_parallel <= 90.0.",
-            "type": "number"
-          },
-          "straight_vertical_longitude_from_pole": {
-            "description": "The longitude (degrees_east) to be oriented straight up from the North or South Pole. Domain: -180.0 <= straight_vertical_longitude_from_pole < 180.0.",
-            "type": "number"
-          },
-          "sweep_angle_axis": {
-            "description": "Indicates the axis on which the view sweeps in a hypothetical gimbal view model of the Earth, as used in the geostationary grid mapping. It corresponds to the outer-gimbal axis of the gimbal view model, about whose axis of rotation the gimbal-gimbal axis moves. This value can adopt two values, x or y, corresponding with the Earth’s E-W or N-S axis, respectively. The counterpart to this attribute is fixed_angle_axis. If set to x, fixed_angle_axis is y, and vice versa. If one of the attributes fixed_angle_axis or sweep_angle_axis is provided, the other is not mandatory, as they can be used to infer each other.",
-            "$ref": "#/$defs/AngleAxisEnum"
-          },
-          "towgs84": {
-            "description": "This indicates a list of up to 7 Bursa Wolf transformation parameters., which can be used to approximate a transformation from the horizontal datum to the WGS84 datum. More precise datum transformations can be done with datum shift grids. Represented as a double-precision array, with 3, 6 or 7 values (if there are less than 7 values the remaining are considered to be zero). Corresponds to a OGC WKT TOWGS84 node.",
-            "type": "number"
-          }
-        },
-        "required": ["Grid_Mapping_Dataset_Name", "grid_mapping_name"]
-      }
-    },
```

Comment:
Looks like a big change to the schema. The biggest contribution to the lines deleted are the attributes allowed in the `Grid_Mapping_Data` section, which are a limited set of attributes, whereas the `CFOverrides` (or `MetadataOverrides`?) cover a much broader array of metadata attributes, so it becomes untenable to define all the options for attributes there.

Reply:
Are you losing any of that information? or was most of it copied from somewhere else in the first place?  I'm ok with it all going since it can be added as a MetadataOverrides object.

Reply:
All the specific attributes definitions were just copied from the CF Conventions page itself.